setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name="octopus-python-client",
-    version="1.3.9",
+    version="1.3.10",
```

Comment:
can we have some mechanism to auto-increment the number here?

Reply:
found https://stackoverflow.com/questions/6786555/automatic-version-number-both-in-setup-py-setuptools-and-source-code but kind of overkill